Docker/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:22.04
+
+LABEL author="OPERA ADT" \
+      description="RTC interface release" \
+      version="interface_0.1"
+
+RUN apt-get -y update &&\
+    apt-get -y install curl git zip make &&\
+    adduser --disabled-password rtc_user
+
+RUN mkdir -p /home/rtc_user/OPERA/RTC
+COPY . /home/rtc_user/OPERA/RTC
+RUN chown -R rtc_user:rtc_user /home/rtc_user/OPERA
+USER rtc_user 
+
+ENV CONDA_PREFIX=/home/rtc_user/miniconda3
+
+WORKDIR /home/rtc_user
+RUN curl -sSL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh &&\
+    bash miniconda.sh -b -p ${CONDA_PREFIX} &&\
+    rm ${HOME}/miniconda.sh
+
+ENV PATH=${CONDA_PREFIX}/bin:${PATH}
+ENV POSTFIX_ISCE3="rtc"
+ENV ISCEHOME=/home/rtc_user
+
+RUN ${CONDA_PREFIX}/bin/conda init bash
+
+#Install ISCE3 from source code
+RUN echo "Installing Conda environment for ISCE3" &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/src &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/build &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/install &&\
+    cd ${ISCEHOME} &&\
+    curl -sSL https://github.com/gshiroma/isce3/archive/refs/heads/opera_rtc.zip -o isce3_opera_rtc.zip &&\
```

Comment:
@gshiroma why do you use this specific fork for the delivery? We should be able to easily trace back the delivered RTC software to upstream ISCE3 versions. For compass delivery, we pushed develop branch out, made a release and then made the conda release and then COMPASS uses that release. We should really do the same.

Reply:
Hi @hfattahi , thank you for your review. We need a branch from ISCE3 that is currently under review (PR 782, github-fn), but it was not yet merged. That PR is used to compute the radar geometry angles over geogrid coordinates.